### ISSUE TITLE
Why should pv_rcnn_ema be train mode(not eval()) in pv_rcnn_ssl.py?  

### ISSUE THREAD

**Issue by honggyuchoi** [THU17cyz/3DIoUMatch-PVRCNN#6]
Thanks for your great works!

In pv_rcnn_ssl.py, 

line 53        ` # self.pv_rcnn_ema.eval()  # Important! must be in train mode`

why should pv_rcnn_ema model be train mode? 

Some layer works different at train mode and eval mode(for example, dropout layer, batch norm layer)

**Comment by yezhen17** [comment]
Honestly, I'm not sure. This observation is an empirical conclusion, and the mechanism is beyond the scope of this project.

I would like to refer you to these two papers (though not directly related):

- Momentum^2 Teacher https://arxiv.org/abs/2101.07525
- Exponential Moving Average Normalization https://arxiv.org/abs/2101.08482

**Comment by honggyuchoi** [comment]
Thanks for super fast reply!
